### ISSUE TITLE
Servlet 6.1 : Refresh with official schema versions

### ISSUE THREAD

**Issue by pmd1nh** [OpenLiberty/open-liberty#26964]
Initial works while waiting for the 6.1 schema to be released (currently unavailable)

Updates to these are expected:
- jakartaee_11.xsd
- web-app_6_0.xsd
- web-fragment_6_0.xsd
- web-common_6_0.xsd

The updated schemas should be published at:
https://jakarta.ee/xml/ns/jakartaee/

In particular:
https://jakarta.ee/xml/ns/jakartaee/#11

**Comment by tbitonti** [comment]
Code refs:

```
dev/com.ibm.ws.javaee.dd/resources/com/ibm/ws/javaee/dd/schemas
dev/com.ibm.ws.javaee.ddmodel/test/com/ibm/ws/javaee/ddmodel/web
dev/com.ibm.ws.javaee.ddmodel/src/com/ibm/ws/javaee/ddmodel/web
dev/repos-pub/ol-baw/dev/com.ibm.ws.javaee.ddmodel_fat
```

**Comment by pmd1nh** [comment]
@tbitonti Hi Tom, we are closing in for the completion of Servlet 6.1.  Can you please check see if there is any action for this issue and close it out?  Thanks!

**Comment by pnicolucci** [comment]
@pmd1nh  new schemas should be coming for any specifications that have them: https://www.eclipse.org/lists/jakartaee-platform-dev/msg04443.html we should keep an eye out for them!

**Comment by tbitonti** [comment]
This is notable (from https://www.eclipse.org/lists/jakartaee-platform-dev/msg04443.html):

> At today's platform call, an inconsistency in our policy for XML schemas regarding version numbers was surfaced:
> 
> Component specifications for EE releases have not always been updated every time a major release of EE 11 has happened.
> 
> To address this inconsistency, for EE 11, we are adopting the following rule:
> 
> **When the version of the spec is updated, the version number and file name of the schemas must be updated to match the version number of the component specification for that EE release, even if there are no normative changes to the XSD files themselves.**
> 
> This rule is added to our existing, common-sense, but apparently not explicitly stated rule:
> 
> **If an XSD file has normative changes during an EE release cycle, its version number and file name must be updated to match the version number of the component specification for that EE release.**
>

**Comment by pnicolucci** [comment]
@tbitonti the latest schemas for Servlet are available now: https://jakarta.ee/xml/ns/jakartaee/#11 Let me know if you have any questions!

**Comment by pmd1nh** [comment]
As part of Concurrent-3.1 DD, the following PRs updated the EE11 schema, including Servlet 6.1 :

https://github.com/OpenLiberty/open-liberty/pull/27221

and

https://github.dev/OpenLiberty/open-liberty/pull/27458

Updated files that related to Servlet 6.1 (and correspondent to the [Jakarta EE11 schema](https://jakarta.ee/xml/ns/jakartaee/#11) for servlet )


1. /open-liberty/dev/com.ibm.ws.javaee.dd/resources/com/ibm/ws/javaee/dd/schemas:

    web-app_6.1.xsd
    https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.javaee.dd/resources/com/ibm/ws/javaee/dd/schemas/web-app_6_1.xsd

    web-commond_6_1.xsd
    https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.javaee.dd/resources/com/ibm/ws/javaee/dd/schemas/web-common_6_1.xsd
    
    web-fragment_6.1.xsd    
    https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.javaee.dd/resources/com/ibm/ws/javaee/dd/schemas/web-app_6_1.xsd


2. /open-liberty/dev/com.ibm.ws.javaee.dd/src/com/ibm/ws/javaee/dd/web

    WebApp.java
    https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.javaee.dd/src/com/ibm/ws/javaee/dd/web/WebApp.java

    WebFragment.java
    https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.javaee.dd/src/com/ibm/ws/javaee/dd/web/WebFragment.java

WebContainer also has a test that has been running with 6.1 web.xml
https://github.com/OpenLiberty/open-liberty/pull/27578/files#diff-531ac2b97dfdcd78dc404f3853b2c6b165617e8223d6ef0a5d1c32afbce0d8e1

I think Servlet 6.1 Schema is good.

@tbitonti can you please verify my above information?  We can close this out if there is no further action/work.

Thanks!

**Comment by tbitonti** [comment]
Looks to be resolved by pull request #27458.  That PR looks good.  Closing.

**Comment by pmd1nh** [comment]
Reopen to update the copyright header for servlet schemas
